packages/frontend/src/pages/ProjectSettings.styles.tsx

```diff
@@ -44,6 +44,7 @@ export const ProjectConnectionContainer = styled.div`
 export const ButtonsWrapper = styled.div`
     display: flex;
     justify-content: flex-end;
+    gap: 10px;
```

Comment:
@IrakliJani are we using this property? Is it supported?

Reply:
we are using it a lot, noone complained about weird layouts so we should be good 😆

Reply:
Perfect. It's my favourite css property of the year